### PR TITLE
added dummy func

### PR DESCRIPTION
**Pull request by gmuser22222**
My pull request.

### CODE REVIEW

**Comment by gmuser11111** [comment]
gmuser11111 comment.

File: src/grasp_monitor/main.py [hunk 1]
```diff
@@ -19,6 +19,10 @@
 from argparse import ArgumentParser, Namespace
 
 
+def dummy_func():
```
Comment (gmuser11111) [inline src/grasp_monitor/main.py:22]: Is this okay?

Comment (gmuser22222) [reply src/grasp_monitor/main.py:22]: Yes, it is.

**Review by gmuser11111 — APPROVED** [review]
All done.

**Pull request merged**